packages/utilities/src/array.ts

```diff
@@ -70,3 +70,14 @@ export function toMatrix<T>(items: T[], columnCount: number): T[][] {
     return rows;
   }, [] as T[][]);
 }
+
+/**
+ * Given an array, it returns a new array that does not contain the item at the given index.
+ * @param array - The array to operate on
+ * @param index - The index of the element to remove
+ */
+export function removeElement<T>(array: T[], index: number): T[] {
+  let cloneArray = array.slice();
```

Comment:
Would it be faster to just do filter?

return array.filter((item, i) => index === i);

looks nicer at least :)